README.md

```diff
@@ -48,12 +48,19 @@ Tip: in case you often use the same non-default values, you can create an alias
 
 ## First time use
 
-- In `sage.py`, set paths to `flexfringe/` executable, and `path_to_ini` variable.
-- A sample alert file is provided with the name `sample-input.json` (T5 alerts from CPTC-2018) to test SAGE. Use the following command: 
+- Clone [FlexFringe repository](https://github.com/tudelft-cda-lab/FlexFringe).
+- Move `spdfa-config.ini` file to `FlexFringe/ini/` directory. Alternatively, you can set the `path_to_ini` variable in `sage.py` to `"./spdfa-config.ini"`.
+- In case you move the `FlexFringe/` directory to another location, update the function `flexfringe` in `model_learning.py` accordingly.
+- You can find the compressed alerts for the [Collegiate Penetration Testing Competition (CPTC)](https://cp.tc/research) and [Collegiate Cyber Defense Competition (CCDC)](https://github.com/FrankHassanabad/suricata-sample-data) datasets (taken from the linked sources) in the `alerts/` directory. To uncompress the alerts, run:
```

Comment:
Referencing the alerts + command that uncompresses them. Does it look fine to you?